src/smart-components/access-management/UsersTable.js

```diff
@@ -0,0 +1,124 @@
+
+const columns = {
+  username: 'Username',
+  email: 'Email',
+  firstName: 'First name',
+  lastName: 'Last name',
+  status: 'Status',
+  userGroup: 'User group',
+  orgAdmin: 'Org admin',
+};
+
+const mockUsers = [
+  {
+    username: 'john.doe',
+    email: 'john.doe@example.com',
+    firstName: 'John',
+    lastName: 'Doe',
+    status: 'Active',
+    userGroup: 'Admin',
+    orgAdmin: true,
+  },
+  {
+    username: 'jane.smith',
+    email: 'jane.smith@example.com',
+    firstName: 'Jane',
+    lastName: 'Smith',
+    status: 'Inactive',
+    userGroup: 'User',
+    orgAdmin: false,
+  },
+];
+
+const addMockUsers = (count) => {
+  for (let i = 0; i < count; i++) {
+    mockUsers.push({
+      username: `user${mockUsers.length + i}`,
+      email: `user@example.com`,
+      firstName: `Random`,
+      lastName: `Person`,
+      status: 'Active',
+      userGroup: 'User',
+      orgAdmin: false,
+    });
+  }
+};
+
+addMockUsers(100);
+
+const perPageOptions = [
+  { title: '5', value: 5 },
+  { title: '10', value: 10 },
+];
+
+const UsersTable = () => {
+  const pagination = useDataViewPagination({
+    perPage: 5,
+  });
+  const { page, perPage } = pagination;
+
+  const selection = useDataViewSelection({
+    matchOption: (selected, item) => selected.username === item.username,
+    initialSelected: [],
+  });
+  const { selected, onSelect, isSelected } = selection;
+
+  const pageData = useMemo(() => mockUsers.slice((page - 1) * perPage, page * perPage), [page, perPage]);
+
+  const handleBulkSelect = (isSelecting) => {
+    onSelect(isSelecting, pageData);
+  };
+
+  return (
+    <DataView>
+      <DataViewToolbar
+        bulkSelect={
+          <BulkSelect
+            count={mockUsers.length}
+            items={pageData}
+            isAllSelected={pageData.every(isSelected)}
+            isParentSelected={pageData.some(isSelected)}
+            onSelectAll={handleBulkSelect}
+          />
+        }
+        pagination={<Pagination perPageOptions={perPageOptions} itemCount={mockUsers.length} {...pagination} />}
+      />
+      <Table aria-label="Users table" ouiaId="UsersTable" variant="compact">
+        <Thead>
+          <Tr>
+            <Th />
+            {Object.values(columns).map((column, index) => (
+              <Th key={index}>{column}</Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody>
+          {pageData.map((user) => (
+            <Tr key={user.username}>
+              <Td
+                select={{
+                  rowIndex: pageData.indexOf(user),
+                  onSelect: (_event, isSelecting) => onSelect(isSelecting, user),
+                  isSelected: isSelected(user),
+                }}
+              />
+              {Object.keys(columns).map((column) => (
+                <Td key={column}>{column === 'orgAdmin' ? (user[column] ? 'Yes' : 'No') : user[column]}</Td>
+              ))}
+            </Tr>
+          ))}
+        </Tbody>
+      </Table>
```

Comment:
This can be replaced with `DataViewTable` https://react-data-view-pr-data-view-14.surge.sh/extensions/data-view/components

Reply:
I'm having some trouble making those rows selectable and adding the action menu on the right. Does the `DataViewTable` support that yet?

Reply:
@CodyWMitchell here is a row selection example https://react-data-view-pr-data-view-14.surge.sh/extensions/data-view/functionality?page=1&perPage=5#selection-example

Reply:
I've just added also an example for row actions - for now it works the same as with regular PF table https://react-data-view-pr-data-view-20.surge.sh/extensions/data-view/components#data-view-table